src/app/examples/list/no-shape/list-no-shape-example.component.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../../../kirby/scss/utils";
+
+kirby-card {
+  margin: 5px;
+  width: 330px;
```

Comment:
Consider calculating from `$breakpoints` map in `src\kirby\scss\base\_variables.scss`

Reply:
Now using map-get($breakpoints, ...) instead. Is this what you meant?